composer/trainer/trainer.py

```diff
@@ -190,9 +195,24 @@ def __init__(
         # self._use_grad_scaling() will raise a RuntimeError if grad scaling is not available when it is required
         warnings.filterwarnings(action="ignore", message="torch.cuda.amp.GradScaler")
 
+        # ScaleSchedule is a deprecated algorithm, but if it is used, updated SSR with its ratio.
+        # TODO(#434): Remove this completely.
+        for algorithm in algorithms or []:
+            if isinstance(algorithm, ScaleSchedule):
+                scale_schedule_ratio = algorithm.ratio
```

Comment:
Would it make sense to rip out the ScaleSchedule algorithm from the algorithms list?

i.e. `algorithms.remove(algorithm)`?